t/page/download.rb

```diff
@@ -23,7 +23,7 @@
       end
 
       it 'should be at rawgit' do
-        subject.download_url.must_include 'cdn.rawgit.com'
+        subject.download_url.must_equal index_at_known_sha.index_url
```

Comment:
If this is now longer actually at rawgit, this is now a rather confusingly named test, that I suspect someone will stumble over at some point. At the very least we should rename this test, but I'm wondering if it's even needed at all. If this test started failing, what would it mean?